service/project.go

```diff
@@ -211,6 +219,147 @@ func (s *ProjectService) DeleteEnvironment(ctx context.Context, projectID, envID
 	return s.envRepo.Delete(ctx, envID)
 }
 
+func (s *ProjectService) CreateInvitation(ctx context.Context, c model.ProjectInvitationCreation, authUserID uuid.UUID) (model.ProjectInvitation, error) {
```

Comment:
> Since invitation has a separate table, separate models on all layers and a non-trivial logic, consider creating a separate service for it.

It's true that `ProjectService` is getting too big. I moved it to `ProjectMembershipService` - this service will manage project invitations and members.

> Or can there be another kind of invitation?

In the future we might add functionality to invite admin users to the app (not specific project). Currently, user has to sign up first and then admin role is assigned after. Which is not the best UX, but good enough for MVP.

Therefore I use route `/projects/invitations/accept|reject` (not just /invitations/..)